lib/Test2/Harness/IPC.pm

```diff
@@ -18,7 +18,7 @@ BEGIN {
     my %SIG_MAP;
     my @SIGNAMES = split /\s+/, $Config{sig_name};
     my @SIGNUMS  = split /\s+/, $Config{sig_num};
-    while (@SIGNAMES || @SIGNUMS) {
+    while (@SIGNAMES) {
```

Comment:
I understand that you want to empty both arrays... but if SIGNAMES is empty and SIGNUM is not...
you are going to use undef as hash key... and this would be a total other mess...
I think we can rely on Config being sane